src/SensioLabs/Melody/Exception/AuthenticationRequiredException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SensioLabs\Melody\Exception;
+
+use SensioLabs\Melody\Handler\ResourceHandlerInterface;
+
+/**
+ * Class AuthenticationRequiredException.
```

Comment:
Eh eh. I knew you'd like it :smile: I simply used it for consistency with the current other classes. But I'll remove this happily !